src/test/java/com/salesforce/dynamodbv2/testsupport/DefaultTestSetup.java

```diff
@@ -164,49 +167,45 @@ public void setupTableData(AmazonDynamoDB amazonDynamoDb,
     }
 
     private List<CreateTableRequest> getCreateRequests(ScalarAttributeType hashKeyAttrType) {
-        return ImmutableList.of(
-            new CreateTableRequest()
```

Comment:
I added a 5th virtual table, so I switched over to using the `CreateTableRequestBuilder` to minimize duplicate code.